charts/metallb/templates/servicemonitor.yaml

```diff
@@ -58,14 +58,6 @@ spec:
   selector:
     matchLabels:
       name: {{ template "metallb.fullname" . }}-speaker-monitor-service
-{{- if .Values.prometheus.serviceMonitor.metricRelabelings }}
```

Comment:
Yes, `metricRelabelings` and `relabelings` are ServiceMonitor `.spec.endpoints` property and not .spec fields itself.